x11-misc/notify-osd/files/notify-osd-0.9.34-fix-integer-conversion.patch

```diff
@@ -0,0 +1,24 @@
+Clang16 will not allow implicit pointer to integer 
+conversions by default. (-Werror=int-conversion by default)
+This patch fixes the pointer to integer conversion.
+
+Bug: https://bugs.gentoo.org/879035
+Patch is upstreamed here: https://answers.launchpad.net/notify-osd/+question/704024
+
+Signed-off-by: Pascal Jäger <pascal.jaeger@leimstift.de>
+--- a/src/bubble-window.c
++++ b/src/bubble-window.c
+@@ -78,8 +78,8 @@ bubble_window_get_accessible (GtkWidget *widget)
+     {
+       AtkObjectFactory *factory = NULL;
+       AtkRegistry *registry = NULL;
+-      GType derived_type = NULL;
+-      GType derived_atk_type = NULL;
++      GType derived_type;
```

Comment:
Is this leaving it uninitialised or does it get handled later on?

Reply:
I don’t think it will make any difference but I can initialize it with 0 instead if we can sleep better than.

Reply:
I'd sleep better with 0 :)

Reply:
Yeah, me too :D